src/pages/shared/components/AutoKeyAddConsent.tsx

```diff
@@ -27,7 +27,7 @@ export const AutoKeyAddConsent = ({ onAccept, onDecline, intent }: Props) => {
         {t(consentMessage[intent])}{' '}
         <a
           hidden
```

Comment:
This link is more of "how does the extension does automatic key addition". It's hidden as I saw there's no explainer for that yet. Maybe can add a page to GitHub repo in extension here for this, but for now, it's hidden.